test/lit.cfg

```diff
@@ -151,3 +151,8 @@ else:
 # POSIX runtime feature
 if config.enable_posix_runtime:
   config.available_features.add('posix-runtime')
+
```

Comment:
So I should add them, or replace `darwin` and `linux` with `not-*`? At this point, I am prone for the latter.